src/main/java/org/sonarlint/intellij/SonarLintIntelliJClient.kt

```diff
@@ -123,6 +111,19 @@ import org.sonarsource.sonarlint.core.rpc.protocol.common.FlowDto
 import org.sonarsource.sonarlint.core.rpc.protocol.common.TextRangeDto
 import org.sonarsource.sonarlint.core.rpc.protocol.common.TokenDto
 import org.sonarsource.sonarlint.core.rpc.protocol.common.UsernamePasswordDto
+import java.io.ByteArrayInputStream
+import java.net.Authenticator
+import java.net.InetSocketAddress
+import java.net.Proxy
+import java.net.URI
+import java.net.URL
+import java.nio.file.Path
+import java.nio.file.Paths
+import java.security.cert.CertificateException
+import java.security.cert.CertificateFactory
+import java.security.cert.X509Certificate
+import java.util.UUID
+import java.util.concurrent.CancellationException
```

Comment:
As discussed, probably a wrong configuration, those imports should not move

Reply:
It depends, this is also a point that was raised by Julien at some point. In kotlin, the configuration for imports puts `java.*` and `javax.*` package separately. In Java it is ordered alphabetically, in Kotlin they are put at the bottom. It depends on how we want to align, I think Eray has the default ordering (java package are separated).

Reply:
I have the same but I think the default one is this:
![image](https://github.com/SonarSource/sonarlint-intellij/assets/14952624/25e5e6c6-9f24-4b3f-a1a9-d230b26d944b)